doc/nrf/ug_matter_creating_accessory.rst

```diff
@@ -68,45 +100,63 @@ To edit clusters using the ZAP tool, complete the following steps:
 
       Zigbee Cluster Configurator window in ZAP tool
 
-#. In the ZAP tool's :guilabel:`Create New Endpoint` menu, create a new endpoint that represents the sensor device:
+   By default, the window displays all available clusters.
+   These can be filtered to show :guilabel:`Only Enabled` clusters.
+   At this stage, only one endpoint is available (Endpoint 0).
+#. In the ZAP tool, click :guilabel:`ADD NEW ENDPOINT`.
+#. In the :guilabel:`Create New Endpoint` menu, create a new endpoint that represents the temperature sensor device type:
 
    .. figure:: images/matter_create_accessory_create_new_endpoint.png
       :alt: Create New Endpoint menu in ZAP tool
 
       Create New Endpoint menu in ZAP tool
 
-#. Configure the following clusters required for this endpoint:
+   The new cluster is created with both the Basic and Identify clusters enabled.
+#. Disable the Basic and Identify clusters by changing their option to ``Not Enabled`` in the drop-down menu in the :guilabel:`Options` column.
```

Comment:
Actually, let's drop this step. Leaving unused clusters enabled shouldn't be harmful and in fact, they are required by the device type specification.